src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Excavator extends Machine {
```

Comment:
The Excavator class is declared as abstract, but given that it provides implementations for all abstract methods from the superclass Machine, it should not be abstract. Abstract classes are meant to be subclasses that provide implementations for abstract methods.